.npmrc

```diff
@@ -0,0 +1 @@
+save-exact = true
```

Comment:
We should not do this in a lib unless we have a very good reason i.e. newer versions of a specific package break things. In that case we should only fix those.

Reply:
👍